lane.py

```diff
@@ -1214,11 +1217,14 @@ def only_show_ready_deliverable_works(
                 work_model.presentation_ready == True,
             )
 
-        # Only find books the default client can fulfill.
```

Comment:
This wasn't working properly and we didn't catch it because the code wasn't tested. Now that the SimplyE client hides books it can't fulfill, I decided to relax this restriction and only find books that have _some_ delivery mechanism.